src/components/Attachments/AttachmentCarousel/Pager/index.tsx

```diff
@@ -19,30 +20,44 @@ type AttachmentCarouselPagerHandle = {
     setPage: (selectedPage: number) => void;
 };
 
-type PagerItem = {
-    key: string;
-    url: string;
+type Attachment = {
     source: string;
 };
 
 type AttachmentCarouselPagerProps = {
-    items: PagerItem[];
-    renderItem: (props: {item: PagerItem; index: number; isActive: boolean}) => React.ReactNode;
-    initialIndex: number;
+    /**
+     * The attachments to be rendered in the pager.
+     */
+    items: Attachment[];
+    /**
+     * The source of the currently active attachment.
```

Comment:
Please add more context to this comment to explain if it's a URL, a URI, or something else (maybe adding an example would help).